corehq/apps/export/tests/test_export_conversion.py

```diff
@@ -352,37 +357,13 @@ def test_remote_app_conversion(self, _, __):
         table = instance.get_table(MAIN_TABLE)
         index, column = table.get_column(
             [PathNode(name='age')],
-            None,
+            'ExportItem',
             None,
         )
         self.assertIsNotNone(column)
         self.assertEqual(column.label, 'age')
         self.assertTrue(meta.is_remote_app_migration)
 
-    @mock.patch(
-        'corehq.apps.export.utils._is_remote_app_conversion',
-        return_value=True,
-    )
-    def test_remote_app_conversion_with_repeats(self, _, __):
-        instance, meta = self._convert_case_export('remote_app_repeats')
```

Comment:
this was just wrong before. it was interpreting the saved export as a case one (even though it was supposed to be a form). it never really made any difference until we decided to handle columns we can't find differently. i.e. form export columns that couldn't be found are created as CustomUserExportColumns while case ones are added to the inferred schemas. Ah the complexity grows...